src/Analyzers/Core/CodeFixes/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -162,6 +162,13 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                             title = CodeFixesResources.Remove_redundant_assignment;
                         }
 
+                        // Also we want to show "Remove redundant assignment" title in pattern matching, e.g.
+                        // if (obj is SomeType someType) <-- "someType" will be fully removed here
```

Comment:
how are we sure of that?  Can you link to the respective fixer code that always does this?  Thanks!

Reply:
The code is buried somewhere in the analizer. I am not familiar with how operation-based analizers work. Hoever, by looking through test, I didn't find any test, where expected behaviour is replacement of original variable with `_`